### ISSUE TITLE
Ask Chris to provide prioritised list of sources and papers

### ISSUE THREAD

**Issue by dancingsushii** [amosproj/amos2024ss04-building-information-enhancer#41]
## User story
1. As a Backend Developer, 
2. I want / need to know all the resources in the prioritized way and have papers to know waht categories to focus on
3. So that I can focus on more important ones first and derive data categories

## Acceptance criteria
* Reply from Chris
* List of sources
* Papers list

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
